packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -501,4 +501,18 @@ export class SchedulerModel {
 
         return job;
     }
+
+    async getJobStatus(jobId: string) {
+        const jobs = await this.database(SchedulerLogTableName)
+            .where(`job_id`, jobId)
+            .orderBy('scheduled_time', 'desc')
+            .returning('*');
+
+        const job = jobs.sort(
+            (a, b) =>
+                statusOrder.indexOf(a.status) - statusOrder.indexOf(b.status),
+        )[0];
```

Comment:
non-blocking but I think we can order by `status` with a DB query no?

Reply:
ordering on sql query will be doing alphabetical ordering, this is ordering following an array... not sure if you can achieve the same on SQL